tracing/tests/span.rs

```diff
@@ -188,6 +188,7 @@ fn cloning_a_span_calls_clone_span() {
         .run_with_handle();
     with_default(subscriber, || {
         let span = span!(Level::TRACE, "foo");
+        #[allow(clippy::redundant_clone)]
```

Comment:
Can we add a comment here explaining that this is necessary to test the behavior this test is trying to test? Thanks!

Reply:
Ah, yeah sorry I should get in the habit of commenting when overriding those lints.